http-server/apache2-web-server/README.md

```diff
@@ -85,28 +104,28 @@ $ nano web/htdocs/index.html
 Or Download it from this repo:
 
 ```
-$ wget https://raw.githubusercontent.com/sylabs/examples/master/http-server/apache2-web-server/index.html
+$ wget -O httpd/htdocs/index.html https://raw.githubusercontent.com/sylabs/examples/master/http-server/apache2-web-server/index.html
 ```
 
-
 <br>
 
-
-To use this structure, we start up an instance binding our host path into the container:
+Now our directory map should look like this:
 
 ```
-$ singularity instance start \
- -B web/htdocs:/usr/local/apache2/htdocs \
- -B web/logs:/usr/local/apache2/logs \
- httpd.sif httpd
+httpd/
```

Comment:
This should be mentioned as `~/httpd` instead, to avoid confusion on the location of the folder